functions/prepared-check-sync/v1.js

```diff
@@ -5,13 +5,30 @@ const uuid = require('uuid/v4')
 const azureStorageHelper = require('../lib/azure-storage-helper')
 const azureTableService = azureStorageHelper.getPromisifiedAzureTableService()
 const accessArrangementsSqlUtil = require('./access-arrangements-sql-util')
+const sqlUtils = require('../lib/sql-helper')
 const preparedCheckSchemaValidator = require('../lib/prepared-check-schema-validator')
 
 const v1 = {}
 
 v1.process = async function process (context, preparedCheckSyncMessage) {
   const { checkCode } = preparedCheckSyncMessage
   context.log('prepared-check-sync: message received', checkCode)
+  const checks = await sqlUtils.sqlFindChecksByCheckCode(checkCode)
+  await Promise.all(checks.map(async (check) => {
+    await v1.updatePreparedChecks(context, check.checkCode)
```

Comment:
Why `await` in the `map` as well as on the `Promise.all`?

Reply:
@jon-shipley both that await and the Promise.all seem redundant to me so I removed them.